known-imports.yaml

```diff
@@ -21,5 +21,9 @@ whitelist:
     allowed: [filename, named]
     prefix: fixtures
     recursive: true
+    ignore:
+      - ignored1.js
+      - ignoredNested/
+      - /ignoredTopLevel/
```

Comment:
So the configuration syntax is that on a `whitelist` directory spec (in `known-imports.yml`) you can specify an `ignore` option (either a single glob or a list of globs)

The idea is to support a glob syntax similar to `.gitignore` (which is what I find intuitive)

So eg here:
- any file named `ignored1.js` at any level of nesting will be ignored
- any entire directory named `ignoredNested` at any level of nesting will be ignored
- an entire directory named `ignoredTopLevel` will only be ignored if it's at the top level (relative to the whitelisted `directory` we're configuring here)